src/main/java/com.gigssandbox/IO/SplittedString.java

```diff
@@ -0,0 +1,13 @@
+package com.gigssandbox.IO;
+
+public class SplittedString {
```

Comment:
We may have implicit contract about splitted strings in some parts of your code by naming String variable as ```String splittedString = ...```. Creation a small separate class with a few logic is not necessary and may puzzle readers